tests/estimator/acforce/vmc.fast.in.xml

```diff
@@ -496,6 +496,8 @@
   1.25722600000000e+00  0.00000000000000e+00  0.00000000000000e+00  0.00000000000000e+00
 </coefficient>
       </sposet>
+    </sposet_builder>
+    <determinantset>
       <slaterdeterminant>
         <determinant id="det_up" sposet="spo-up" size="5"/>
         <determinant id="det_down" sposet="spo-dn" size="4"/>
```

Comment:
Please remove `id`, `group` and `size` in `determinant`. Also replace `sposet_builder` with `sposet_collection`.

Reply:
Wondering if we catch the case where the user specifies a particleset and a determinant with different sizes?

Reply:
determinant size is derived from particleset. `size` in the old input was for constructing sposet only.

Reply:
Sure, but we should catch inconsistencies in the input. Do we read size anymore? Ideally we would warn the user about unneeded/unused inputs.

Reply:
(Not something to fix here)

Reply:
Once we disallow(warning->error) users creating sposet in determinants. I will issue a warning about all the outdated input.